root/etc/s6-overlay/s6-rc.d/init-unifi-controller-config/run

```diff
@@ -20,6 +20,11 @@ for i in "${symlinks[@]}"; do
 done
 
 if [[ ! -e /config/data/system.properties ]]; then
+    sed -i "s/~MONGO_USER~/${MONGO_USER}/" /defaults/system.properties
+    sed -i "s/~MONGO_PASS~/${MONGO_PASS}/" /defaults/system.properties
+    sed -i "s/~MONGO_HOST~/${MONGO_HOST}/" /defaults/system.properties
+    sed -i "s/~MONGO_PORT~/${MONGO_PORT}/" /defaults/system.properties
+    sed -i "s/~MONGO_DBNAME~/${MONGO_DBNAME}/" /defaults/system.properties
     cp /defaults/system.properties /config/data
 fi
```

Comment:
Worth mentioning in the readme that the variables are only read on a clean install, if we aren't going to modify the live file.

Reply:
It says in the variable description https://github.com/linuxserver/docker-unifi-controller/pull/197/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R193

Reply:
protip: dont read PRs the first 15 minutes after waking up